src/main/kotlin/org/sol4k/Lamport.kt

```diff
@@ -0,0 +1,22 @@
+package org.sol4k
+
+import java.math.BigDecimal
+import java.math.RoundingMode
+
+fun lamportToSol(v: String): BigDecimal =
```

Comment:
Let's wrap all these functions into a `Lamport` (or maybe better `Convertation`) companion object for better Java interoperability. In this current form, Java users will have to call something like `LamportKt. lamportToSol()`. Also let's have them all as `@JvmStatic`